### PR TITLE
Feature/geode 3897

### PR DESCRIPTION
**Pull request by pivotal-eshu**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java [hunk 1]
```diff
@@ -107,6 +94,33 @@ public Object call() throws Exception {
     });
   }
 
+  private static String TEST_REGION_NAME = "testSimplePdx";;
```
Comment (nreich) [inline geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java]: should move this to the top of the class and make it final.

File: geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java [hunk 2]
```diff
@@ -107,6 +94,33 @@ public Object call() throws Exception {
...
+
+  private void createPR() {
+    getCache().createRegionFactory(RegionShortcut.PARTITION)
+        .setPartitionAttributes(new PartitionAttributesFactory<Integer, Object>().create())
+        .create(TEST_REGION_NAME);
+  }
+
+  @Test
+  public void testSimplePutOnPRWitTx() {
```
Comment (nreich) [inline geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java]: Spelling error in method name.

File: geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java (hunk 1 above)
Comment (pivotal-eshu) [reply geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java]: Done.

File: geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java (hunk 2 above)
Comment (pivotal-eshu) [reply geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java]: Fixed.

**Pull request merged**